README.md

```diff
@@ -74,7 +129,11 @@ cd /srv/docker/bridgehead
 sudo ./bridgehead install <PROJECT>
 ```
 
-... and follow the instructions on the screen. You should then be prompted to do the next step:
+When prompted with "Please enter your site", you should enter the Repository Short Name (RSN) for GitLab that you were given in the previous section.
```

Comment:
```suggestion
When prompted with "Please enter your site", you should enter the `SITE_ID` you were given in the previous section.
```

Reply:
I would avoid these steps all together, just clone the repo before the installation using the URL sent via email. No need to complicate it